bin/webpack.js

```diff
@@ -318,7 +318,7 @@
 			});
 
 			if (typeof outputOptions.colors === "undefined")
-				outputOptions.colors = require("supports-color");
+				outputOptions.colors = require("supports-color").stdout;
```

Comment:
Anything changed in a newer version?

Reply:
Doesn't look like anything changed relevant to us other than the API.
There are no changelogs or release notes but you can take a look at this PR [#64](https://github.com/chalk/supports-color/pull/64).

Reply:
Remove the stdout and you're good. Doesn't work with it